ui.py

```diff
@@ -1,4 +1,33 @@
+import math
+
 class Ui:
     def msg(message):
         print(message)
 
+class CommandLinePrompt:
+    def get_input(choice):
+        return input(choice)
+
+class BoardPresenter:
+    def display_terminal_board(current_board_state):
+        cbs = current_board_state
```

Comment:
what does this renaming buy you?

Reply:
The renaming gives me a descriptive name for what this method does. It allows me to potentially add a "display_browser_board" that formats the data in a different way, and keep those two separate.

Reply:
`cbs` is descriptive?